### PR TITLE
make_rob_whole (he was chopped in half before)

### PR DESCRIPTION
**Pull request by iveksl2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,8 +93,8 @@ If you want more git-related aliases for making your git workflow faster, also l
 
 
 ## Got any more plugins to share?
-* [Send.zsh](https://github.com/robertzk/send.zsh), a git command by ro
-bertzk that combines `git add .`, `git commit -a -m`, and `git push ori
+* [Send.zsh](https://github.com/robertzk/send.zsh), a git command by robertzk 
+that combines `git add .`, `git commit -a -m`, and `git push ori
 gin <branch>`.
```
Comment (peterhurford) [inline README.md]: @iveksl2 Can you do the same here?

**Comment by iveksl2** [comment]
@peterhurford done

**Comment by peterhurford** [comment]
@iveksl2 thanks!

**Comment by peterhurford** [comment]
@iveksl2 You have to make this PR against my repo for me to be able to merge it.

**Pull request closed without merging**